### DISCUSSION TITLE
md5 hashed <password><username> in userlist does not work

### DISCUSSION THREAD

**Discussion by ahmedabdou14** [pgbouncer/pgbouncer#885] [Q&A]
Hi, this is an issue i have been trying to solve day and night for the past week.
I am using docker, specifically edoburu/pgbouncer:1.18.0
This is my pgbouncer.ini
```ini
[databases]
ets = host=db port=5432 user=ets

[pgbouncer]

;;;
;;; Administrative settings
;;;
; logfile = pgbouncer.log
; pidfile = pgbouncer.pid

;;;
;;; Where to wait for clients
;;;
;; IP address or * which means all IPs
listen_addr = 0.0.0.0
listen_port = 6432

;;;
;;; Authentication settings
;;;
;; any, trust, plain, md5, cert, hba, pam
auth_type = md5
auth_file = /etc/pgbouncer/userlist.txt

;; When server connection is released back to pool:
;;   session      - after client disconnects (default)
;;   transaction  - after transaction finishes
;;   statement    - after statement finishes
; asyncpg does not support transaction pooling
pool_mode = session

...
```

This is my userlist.txt
```txt
"ets" "md5be32a9578421db00c2579b50f17a2431"
```

The password hashed is md5(`password123ets`) 
where password123 is the password and ets is the username

For some reason, it will not authenticate it! it either gives wrong type of password error or incorrect password.
I am not sure what i am missing exactly and why is it not working.
Instead it authenticates it if i provide the plaintext lol, where i clearly mentioned md5 auth_type in the .ini

**Comment by petere** [comment]
This looks correct at first glance. Please give complete commands and log messages etc. Maybe there is a little detail missed.
